src/parse/asp/config_test.go

```diff
@@ -0,0 +1,21 @@
+package asp
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestResolvePluginTargetValues(t *testing.T) {
+	values := resolvePluginValue([]string{"//path/to:target"}, "path/to/subrepo")
+	assert.Equal(t, []string{"///path/to/subrepo//path/to:target"}, values)
+
+	values = resolvePluginValue([]string{"///path/to/another/plugin//path/to:target"}, "path/to/subrepo")
+	assert.Equal(t, []string{"///path/to/another/plugin//path/to:target"}, values)
+
+	values = resolvePluginValue([]string{"///path/to/another/plugin//path/to:target"}, "")
+	assert.Equal(t, []string{"///path/to/another/plugin//path/to:target"}, values)
+
+	values = resolvePluginValue([]string{"//path/to:target"}, "")
+	assert.Equal(t, []string{"/////path/to:target"}, values)
```

Comment:
Is this what we want?

Reply:
That's how it seems to be meant to work. In general, evaluated labels are resolved fully according to their context (i.e. if they are in a subrepo or not).
So, `resolvePluginValue([]string{"//path/to:target"}, "")` refers to the `//path/to:target` config value in the host repo. Since this is to be used in the plugin, it needs the `///` prefix, otherwise it will get resolved locally to the subrepo.

Example:

Host repo
`.plzconfig` file
```ini
[Plugin "proto"]
LanguageDef = //build_defs:python
```

Proto plugin
`build_defs/BUILD` file
```python
subinclude("//build_defs/sdk")

protoc_plugins(
    name = "proto_languages",
    build_defs = CONFIG.PROTO.DEFINITIONS,
    type = "proto_language",
)
```

If `//build_defs:python` doesn't get the `///` prefix, it will be resolved as something like `///proto//build_defs:python`